vispy/visuals/tests/test_mesh.py

```diff
@@ -31,6 +31,23 @@ def test_mesh_color():
     np.testing.assert_allclose(vertices['position'], new_vertices)
 
 
+@requires_pyopengl()
+@requires_application()
+def test_mesh_with_values():
```

Comment:
```suggestion
def test_mesh_with_vertex_values():
```

Reply:
A good change and thanks for saving me the keystrokes!